recipes/msaprobs/meta.yaml

```diff
@@ -0,0 +1,40 @@
+package:
+  name: msaprobs
+  version: 0.9.7
```

Comment:
You should introduce a Jinja2 variable for the version. At the top of the file, add
```
{% set version = "0.9.7" %}
```
Then you can write here:
```
version: "{{ version }}"
```